app/models/relationship.rb

```diff
@@ -0,0 +1,6 @@
+class Relationship < ApplicationRecord
+  belongs_to :follower, class_name: "User"
+  belongs_to :followed, class_name: "User"
+  validates :follower_id, presence: true
```

Comment:
follower yêu cầu là user đó phải tồn tại còn follower_id thì chỉ kiểm tra có id đó là được.
dùng follwer đảm bảo tính an toàn hơn.